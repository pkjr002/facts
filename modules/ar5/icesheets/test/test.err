ar5_project_icesheets_dev.py:137: DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`. To silence this warning, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  startrate = (startratemean + startratepm * np.array([-1,1], dtype=np.float)) * momm
ar5_project_icesheets_dev.py:141: DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`. To silence this warning, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  acceleration = (np.array(finalrange, dtype=np.float) - startrate * finalyr.mean()) / (finalyr**2).mean()
