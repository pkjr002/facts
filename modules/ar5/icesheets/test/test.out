Initiating test in /scratch/rk509/test.1666784183...

Extracting data files...

Executing workflow...
python ar5_preprocess_icesheets.py --pipeline_id icesheets-ar5-icesheets-ssp585 --scenario ssp585 --baseyear 2005 --tlm_data 1
python ar5_fit_icesheets.py --pipeline_id icesheets-ar5-icesheets-ssp585
python ar5_project_icesheets_dev.py --pipeline_id icesheets-ar5-icesheets-ssp585 --nsamps 1000 --pyear_start 2020 --pyear_end 2300 --pyear_step 10 --seed 4321 --crateyear_start 2080 --crateyear_end 2100
python ar5_postprocess_icesheets_dev.py --pipeline_id icesheets-ar5-icesheets-ssp585 --chunksize 662 --locationfile /home/rk509/facts/modules/ar5/icesheets/test/../../../../scripts/moduletest/location_list.lst

Collecting output files...
icesheets-ar5-icesheets-ssp585_AIS_globalsl.nc
icesheets-ar5-icesheets-ssp585_AIS_localsl.nc
icesheets-ar5-icesheets-ssp585_EAIS_globalsl.nc
icesheets-ar5-icesheets-ssp585_EAIS_localsl.nc
icesheets-ar5-icesheets-ssp585_GIS_globalsl.nc
icesheets-ar5-icesheets-ssp585_GIS_localsl.nc
icesheets-ar5-icesheets-ssp585_WAIS_globalsl.nc
icesheets-ar5-icesheets-ssp585_WAIS_localsl.nc

Cleaning up...
